src/config/config.js

```diff
@@ -1,14 +1,20 @@
-const loginServerRootAddress = 'http://10.58.2.34:8000';
+const loginServerRootAddress = 'http://10.58.4.238:8000';
 
 export const LOGIN_SERVER_ADDRESS = {
   loginMainAddress: `${loginServerRootAddress}/users/check`,
   loginSignInAddress: `${loginServerRootAddress}/users/login`,
   loginSignUpAddress: `${loginServerRootAddress}/users/signup`,
 };
 
+const cartServerRootAddress = 'http://10.58.4.238:8000';
+
+export const CART_SERVER_ADDRESS = {
+  cartMainAddress: `${cartServerRootAddress}/carts`,
+};
+
```

Comment:
* baseURL은 하나인데, 같은걸 반복해서 어려운 이름으로 나눠놓은게 어떤이유인지 알수 있을까요?
* endpoint별로 key만 부여해주면 객체 1개로 관리할 수 있습니다.
* 이름이 좀 장황합니다. 우리는 url주소가 필요하지, 이름이 필요한게 아닙니다.

```suggestion
const BASE_URL = 'http://10.58.4.238:8000';

export const API = {
  carts: `${BASE_URL}/carts`,
};

```

Reply:
지금 백앤드 서버가 하나로 통일이 안된 상황이라 임시로 나눠놨습니다. 팀원들과 해당 내용에 대해서 말해놨고 최종적으로 하나의 BASE URL을 사용하는 식으로 수정하겠습니다.